src/proxy/ddl_test.go

```diff
@@ -80,6 +92,45 @@ func TestProxyDDLDB(t *testing.T) {
 	}
 }
 
+func TestProxyDDLDBError(t *testing.T) {
+	log := xlog.NewStdLog(xlog.Level(xlog.PANIC))
+	fakedbs, proxy, cleanup := MockProxy(log)
+	defer cleanup()
+	address := proxy.Address()
+
+	// create database querys
+	querysError := []string{
+		// keyword error
+		"create database test1 char set default",  // char-->charset
+		"create database test2 character default", // character-->character set
+		// collate_name/charset_name should not be emtpy
+		"create database test4 collate ",
+		"create database test4 charset ",
+		// In Radon, utf8m/latin1 are parsed successfully, we just treat them as
+		// normal strings, but in mysql, they are not correct character set, so
+		// we still treat the sql next are not correct.
+		"create database test3 charset utf8m",  // utf8m-->utf8mb4
+		"create database test4 collate latin1", // latin1-->latin1_swedish_ci
+		// ERROR 1253 (42000): COLLATION 'latin1_swedish_ci' is not valid for CHARACTER SET 'utf8mb4'
```

Comment:
Does the comment refer to the above sql or the following sql？

Reply:
> Does the comment refer to the above sql or the following sql？

Refer to the following sql.